vmdb/app/models/configured_system.rb

```diff
@@ -0,0 +1,7 @@
+class ConfiguredSystem < ActiveRecord::Base
+  include NewWithTypeStiMixin
+  belongs_to :configuration_manager
+  belongs_to :configuration_profile
```

Comment:
Same [comment](https://github.com/ManageIQ/manageiq/pull/947/files#r22804522) goes for this relationship.